pages/common/xplr.md

```diff
@@ -0,0 +1,24 @@
+# xplr
+
+> Explore the file system on your terminal.
+> More information: <https://github.com/sayanarijit/xplr>.
+
+- Open a directory:
+
+`xplr {{directory}}`
+
+- Focus on a file:
+
+`xplr {{file}}`
+
+- Edit file(s):
+
+`vim $(xplr)`
```

Comment:
Hmm, this is a cool example, but I'm not sure if it fits our guidelines 🤔

It may though, so let's wait and see.

Reply:
ping @sbrl.  What do you think about this?

Reply:
Hrm, I'm not sure. Generally we want to demonstrate the functionality of the command itself, rather than the that of other commands and the builtin shell.